install.sh

```diff
@@ -707,7 +707,7 @@ ExecStart=${BIN_DIR}/rke2 \\
     ${CMD_RKE2_EXEC}
 
 EOF
-    echo "HOME=/root" > /etc/systemd/system/rke2.service.env
+    echo "HOME=/root" | ${SUDO} tee /etc/systemd/system/rke2.service.env
```

Comment:
Do we use sudo anywhere else in this script? I think we make it incumbent upon the user/operator to have the correct permissions to execute this script.

Reply:
Ah, searched, we have a SUDO macro of sorts.

Reply:
We use it a quite a few spots throughout the script here and in K3S. I'm with ya in regards to your point though.

Reply:
In k3s we allow a non-root user to run install.sh.

Reply:
LGTM :+1:

Reply:
Just from an audit log perspective, piping stuff into a sudo'd command seems a little sketchy since all you'll see is the tee. Maybe capture the full command by doing:
```bash
${SUDO} /bin/sh -c 'echo HOME=/root > /etc/systemd/system/rke2.service.env'
```

Reply:
I'd be happy to do this in another PR and update the rest of the references at the same time and keep the testing surface area consistent.

Reply:
My two cents - separate PR for that later.